wps-cloudformation-template.yaml

```diff
@@ -71,6 +71,13 @@ Parameters:
     AllowedValues:
       - true
       - false
+  awsRegionSes:
```

Comment:
Home come we need this? Can't we just assume it's in the same region as everything else?

Reply:
Nop... SES is not available in all regions.

Reply:
Ah, simple email service in only available in those three regions, this seems like the best solution then